interface/backend/api/views.py

```diff
@@ -179,24 +179,35 @@ def candidates_info(request):
     serialized_candidates = serializers.CandidateSerializer(all_candidates, context={'request': None}, many=True).data
 
     # append DICOM files to response
-    for candidate in serialized_candidates:
-        series = candidate['case']['series']
-
-        if 'files' not in series:
-            # using `glob1` as it returns filenames without a directory path
-            series['files'] = glob.glob1(series['uri'], '*.dcm')
+    [append_files_to_candidate(candidate) for candidate in serialized_candidates]
 
     return Response(serialized_candidates)
 
 
-@api_view(['GET'])
-def candidate_mark(request, candidate_id):
-    return Response({'response': "Candidate {} was marked".format(candidate_id)})
+@api_view(['POST'])
+def review_candidate(request, candidate_id):
+    try:
+        review_result = json.loads(request.body)['review_result']
```

Comment:
@Serhiy-Shekhovtsov For the record, `request.data` does this for you (http://www.django-rest-framework.org/api-guide/requests/#data)

Reply:
@isms thanks for sharing. I can see you've updated the code to support **partial_update**.
Btw, the `append_files_to_candidate` is rather a workaround. We should load all files for selected case only once at case start.